nitro-root-enclave/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
```

Comment:
Can we have the standard licensing/copyright header added to the `Cargo.toml` and the authors field standardised on "The Veracruz Development team" and the "about" tag also filled in?